src/olympia/reviewers/templates/reviewers/addon_details_box.html

```diff
@@ -88,7 +88,33 @@ <h2>{{ addon_name }}</h2>
           {# XXX future feature: 'Localizations' ... #}
           <tr class="meta-rating">
             <th>{{ _('Rating') }}</th>
-            <td>{{ reviews_link(addon) }}</td>
+            <td>
+              <p class="addon-rating">
+                {% with num=addon.total_ratings %}
+                  {% if num %}
+                  {{ addon.average_rating|float|stars }}
+                  {% with count='<span itemprop="count">{0}</span>'|format_html(num|numberfmt) %}
+                    {# Using num=count so we don't change an L10n string. #}
+                    {% if not addon.is_disabled or action_allowed(amo.permissions.RATINGS_MODERATE) %}
+                      {% set review_url = url('addons.ratings.list', addon.slug) %}
+                      {% if addon.is_disabled %}
```

Comment:
we want this always when the user has the permission

Reply:
see the original comment in the issue - the two alternatives are having the permission and not.

Reply:
We want to only linkify if the user has permission `Ratings:Moderate`, otherwise leave it unlinked. Is it right ?